internal/pkg/service/buffer/storage/file/manager.go

```diff
@@ -108,7 +109,13 @@ func (m *Manager) createFile(ctx context.Context, rb rollback.Builder, mapping m
 	fileName := file.Filename()
 	slice := model.NewSlice(file.FileKey, now, mapping, 1, nil)
 
-	resource, err := storageapi.CreateFileResourceRequest(&storageapi.File{Name: fileName, IsSliced: true}).Send(ctx, m.client)
+	resource, err := storageapi.
+		CreateFileResourceRequest(&storageapi.File{
+			Name:     fileName,
+			IsSliced: true,
+			Tags:     []string{fmt.Sprintf("buffer.exportID=%s", mapping.ExportID.String()), fmt.Sprintf("buffer.receiverID=%s", mapping.ReceiverID.String())},
```

Comment:
Dal bych tam ještě prefix, třeba s tečkou. 😄 Jmenný konvence zjevně žádný nejsou, viděl jsem akorát to rovnítko u tagů tabulek.